lightning/src/ln/channel.rs

```diff
@@ -356,6 +356,11 @@ pub const FEE_SPIKE_BUFFER_FEE_INCREASE_MULTIPLE: u64 = 2;
 #[cfg(not(fuzzing))]
 const FEE_SPIKE_BUFFER_FEE_INCREASE_MULTIPLE: u64 = 2;
 
+/// If we fail to see a funding transaction confirmed on-chain within this many blocks after the
+/// channel creation on an inbound channel, we simply force-close and move on.
+/// This constant is the one suggested in BOLT 2.
+const FUNDING_CONF_DEADLINE_BLOCKS: u32 = 2016;
```

Comment:
should this be a constant set by us, or ought LDK consumers be able to override it?

Reply:
We could allow users to override it, its in the spec but its not a hard requirement and the fundee has to be the one to pick. That said, I'm not sure if we need to? We should pick a constant high enough that it shouldn't conceivably be an issue for users, and I think two weeks is probably far enough out, no?